examples/next-sse-chat/src/app/channels/[channelId]/chat.tsx

```diff
@@ -31,6 +38,68 @@ const listWithAnd = (list: string[]) => {
   return `${list.slice(0, -1).join(', ')}, and ${list.at(-1)}`;
 };
 
+function SubscriptionStatus(props: {
```

Comment:
cc @BeBoRE I added this component

Reply:
Looks really good, nice showcase of the feature